### ISSUE TITLE
Confidence metrics

### ISSUE THREAD

**Issue by georgkempf** [hpcaitech/FastFold#135]
It looks like that "predicted aligned error" and pTM are not included in the prediction_results dict in case of openfold/fastfold. Is this planned to be added?

**Comment by Gy-Lu** [comment]
Yes, it would be developed with a high priority.

**Comment by kashyapchhatbar** [comment]
A recently merged PR #132 saves the pickle file which contains these information

```python
>>> import pickle
>>> df = pickle.load(open("predicted_results.pkl", "rb"))
>>> df.keys()
```

`dict_keys(['msa', 'pair', 'single', 'sm', 'final_atom_positions', 'final_atom_mask', 'final_affine_tensor', 'lddt_logits', 'plddt', 'distogram_logits', 'masked_msa_logits', 'experimentally_resolved_logits', 'tm_logits', 'predicted_tm_score', 'aligned_confidence_probs', 'predicted_aligned_error', 'max_predicted_aligned_error'])`

**Comment by georgkempf** [comment]
Great thanks!
